docs/install/docker.md

```diff
@@ -60,6 +61,8 @@ services:
   web_recipes:
     image: vabene1111/recipes
     restart: always
+    ports:
+      - 80:8080
```

Comment:
this is not correct. If the web_recipes container already exposts port 80 nginx can no longer do so

Reply:
This makes sense, thank you. I did some further testing and think I've figured it (Docker basics) out.

For example, to expose on port 3000, you could use `docker run -d -p 3000:8080 ...` or, for Docker Compose,
```
  nginx_recipes:
    ports:
    - 3000:80
```

It worked in my tests.